server/graphql/v2/object/Host.js

```diff
@@ -437,6 +452,124 @@ export const Host = new GraphQLObjectType({
           };
         },
       },
+      contributionStats: {
+        type: new GraphQLNonNull(ContributionStats),
+        args: {
+          account: {
+            type: new GraphQLList(new GraphQLNonNull(AccountReferenceInput)),
+            description: 'A collection of accounts for which the contribution stats should be returned.',
+          },
+          dateFrom: {
+            type: GraphQLDateTime,
+            description: 'Calculate contribution statistics beginning from this date.',
+          },
+          dateTo: {
+            type: GraphQLDateTime,
+            description: 'Calculate contribution statistics until this date.',
+          },
+        },
+        async resolve(host, args, req) {
+          if (!req.remoteUser?.isAdmin(host.id)) {
+            throw new Unauthorized('You need to be logged in as an admin of the host to see the contribution stats.');
+          }
+          const where = { HostCollectiveId: host.id, kind: 'CONTRIBUTION' };
+          const dateRange = getFilterDateRange(args.dateFrom, args.dateTo);
+          if (dateRange) {
+            where.createdAt = dateRange;
+          }
+          if (args.account) {
+            const collectiveIds = args.account.map(async account => {
+              const accountObj = await fetchAccountWithReference(account, { throwIfMissing: true });
+              return accountObj.id;
+            });
+            await Promise.all(collectiveIds).then(collectiveIds => {
+              where.CollectiveId = { [Op.in]: collectiveIds };
+            });
+          }
+          const contributionsCount = await models.Transaction.count({
+            where,
+          });
+          const contributionsAmountSum = await models.Transaction.sum('amount', where);
```

Comment:
We're counting both `DEBIT` and `CREDIT` in there, which probably results in incorrect numbers. Same for `contributionsCount`, and probably the other stats too.

Reply:
What do we want to achieve there? Counting total contributions amount is a bit unclear to me?

When we're doing such things in other places, the query is usually **way** more complicated.

Reply:
@Betree : For this one, I've made the contributions query consider only `CREDITS` and the expense queries only consider `DEBITS`. Does that make sense? What I was thinking is that for contributions we are interested in the amounts that came into the collective whereas for expenses it's the other way around. 🤔 

@znarf : The idea here is to show the total contributions and expenses for all the collectives hosted by a given host. My understanding is that we need the total money in and the total money out (correct me if I am wrong @Betree). 🤔 

![image](https://user-images.githubusercontent.com/12435965/134424561-ad6c077f-f2c0-4e54-b4da-a656b4b9bf69.png)